packages/page-experience/lib/checks/fontDisplay.js

```diff
@@ -14,6 +14,19 @@
  * limitations under the License.
  */
 
+// Copy of https://github.com/ampproject/amp-toolbox/blob/main/packages/linter/src/rules/NoIconFontIsUsed.ts#L9
+const ICON_FONTS = new Set([
+  'FontAwesome',
+  'Font Awesome',
+  'NerdFontsSymbols',
+  'Nerd Font',
+  'Material Icons',
+  'IcoFont',
+  'icon',
+  'icons',
+  'icomoon',
```

Comment:
There are a couple other icon fonts that are common in WordPress:

```suggestion
  'icomoon',
  'Dashicons',
  'Genericons',
```

Reply:
Added these to the list